code/modules/mob/dead/observer/observer.dm

```diff
@@ -495,13 +495,18 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	updateghostimages()
 
 /proc/updateallghostimages()
+	remove_nulls(ghost_images_full)
```

Comment:
holy shitballs this is a thing? i'm going to have to apply this, does it pick up "" too or just null